assets/app/scripts/filters/resources.js

```diff
@@ -24,7 +24,8 @@ angular.module('openshiftConsole')
       "encodedDeploymentConfig": ["openshift.io/encoded-deployment-config"],
       "deploymentVersion": ["openshift.io/deployment-config.latest-version"],
       "displayName": ["openshift.io/display-name"],
-      "description": ["openshift.io/description"]
+      "description": ["openshift.io/description"],
+      "buildNumber": ["openshift.io/build.number"]
```

Comment:
upstream pattern we've been following is `openshift.io/` prefix, prefer hyphenated names with a `.` field separator